packages/cli/src/loaders/index.spec.ts

```diff
@@ -859,6 +859,175 @@ user.password=Contraseña
       );
     });
   });
+  describe('xliff bucket loader', () => {
+    it('should load xliff data', async () => {
+      setupFileMocks();
+  
+      const input = `
+  <xliff xmlns="urn:oasis:names:tc:xliff:document:2.0" version="2.0" srcLang="en-US">
+    <file id="namespace1">
+      <unit id="key1">
+        <segment>
+          <source>Hello</source>
+        </segment>
+      </unit>
+      <unit id="key2">
+        <segment>
+          <source>An application to manipulate and process XLIFF documents</source>
+        </segment>
+      </unit>
+      <unit id="key.nested">
+        <segment>
+          <source>XLIFF Data Manager</source>
+        </segment>
+      </unit>
+      <group id="group">
+        <unit id="groupUnit">
+          <segment>
+            <source>Group</source>
+          </segment>
+        </unit>
+      </group>
+    </file>
+  </xliff>
+      `.trim();
+      
+      const expectedOutput =  {
```

Comment:
expected output must be flat

Reply:
Examples:

messages.en.xlf
```
<?xml version="1.0" encoding="UTF-8"?>
<xliff version="1.2" xmlns="urn:oasis:names:tc:xliff:document:1.2">
  <file source-language="en" datatype="plaintext" original="messages">
    <body>
      <trans-unit id="nav.home">
        <source>Home</source>
      </trans-unit>
      <trans-unit id="nav.about">
        <source>About</source>
      </trans-unit>
      <trans-unit id="greeting">
        <source>Hello, {username}!</source>
      </trans-unit>
      <trans-unit id="items.count">
        <source>{count} items in your cart</source>
      </trans-unit>
      <trans-unit id="checkout.button">
        <source>Proceed to Checkout</source>
      </trans-unit>
    </body>
  </file>
</xliff>
```

messages.es.xlf
```
<?xml version="1.0" encoding="UTF-8"?>
<xliff version="1.2" xmlns="urn:oasis:names:tc:xliff:document:1.2">
  <file source-language="en" target-language="es" datatype="plaintext" original="messages">
    <body>
      <trans-unit id="nav.home">
        <source>Home</source>
        <target>Inicio</target>
      </trans-unit>
      <trans-unit id="nav.about">
        <source>About</source>
        <target>Acerca de</target>
      </trans-unit>
      <trans-unit id="greeting">
        <source>Hello, {username}!</source>
        <target>¡Hola, {username}!</target>
      </trans-unit>
      <trans-unit id="items.count">
        <source>{count} items in your cart</source>
        <target>{count} artículos en tu carrito</target>
      </trans-unit>
      <trans-unit id="checkout.button">
        <source>Proceed to Checkout</source>
        <target>Proceder al pago</target>
      </trans-unit>
    </body>
  </file>
</xliff>
```

flattened json equivalent:

```
{
  "nav.home": "Home",
  "nav.about": "About",
  "greeting": "Hello, {username}!",
  "items.count": "{count} items in your cart",
  "checkout.button": "Proceed to Checkout"
}
```